vtysh/bgp_vty.c

```diff
@@ -2708,6 +2708,30 @@ is_ibgp_peer(const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor,
   return false;
 }
 
+void
+remove_parameters_invalid_for_ibgp_peer(const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor)
+{
+    if (ovs_bgp_neighbor->remove_private_as) {
+        ovsrec_bgp_neighbor_set_remove_private_as(ovs_bgp_neighbor, NULL, 0);
+    }
+
+    if (ovs_bgp_neighbor->local_as) {
```

Comment:
Similarly suggest instead of operation with a pointer used with a counter operation.,